x-pack/test/plugin_api_integration/plugins/task_manager/init_routes.js

```diff
@@ -91,7 +94,7 @@ export function initRoutes(server, taskTestingEvents) {
         payload: Joi.object({
           task: Joi.object({
             taskType: Joi.string().required(),
-            interval: Joi.string().optional(),
+            schedule: Joi.string().optional(),
```

Comment:
```suggestion
            schedule: Joi.object({
              interval: Joi.string(),
            }).optional(),
```

Reply:
eek why wasn't that caught by functional test?? Investigating

Reply:
ahhh because we don't send a new interval on the ensure api call.... that's why.

Reply:
We don't need that in that API call (our type ensures this is in the actual TM) in our FTs so this'll be removed